### DISCUSSION TITLE
Meade LX200 EMC Stellarium Control Problem

### DISCUSSION THREAD

**Discussion by jhenley57** [Stellarium/stellarium#2853] [Q&A]
I have a LX200 EMC scope, it is connected to WIN10 PC through COM 1, Using LX200 Compatible in setting, when I press start under telescope control, it says stopped and will not connect. The lx200 does work on Meade Autostar Suite 5.56 and also works with Skytrack software ok.  Any Ideas  downloaded win64 ver 1.1 , all else seems fine just wont connect to scope  tnx

**Comment by LeeBowditch** [comment]
I too have the same problem/issue after upgrading to v1.1 tonight. Uninstalled v1.1 and reinstalled v0.22.1 and works fine again. Redone upgrade and again does not start just shows "stopped" on the telescope config panel. I have a LX200GPS 12".

**Comment by gzotti** [comment]
1) Are those instruments connected directly via LX200 protocol or as ASCOM instruments? 
2) Does Qt5-based version 1.1.1-qt5 work?

**Reply by LeeBowditch** [reply]
My telescope is connected directly via RS-232 on Com1. I am running Stellarium 1.1 on Windows 10.

**Reply by gzotti** [reply]
https://github.com/Stellarium/stellarium/wiki/Common-Problems-for-the-current-version#problems-with-telescopecontrol-plugin

The question was whether you are running the Qt5-based version 1.1.1 (found following the "all releases" link on stellarium.org) or the more modern Qt6-based (default download) version which has reportedly still unsolved problems with the TelescopeControl plugin.

**Reply by LeeBowditch** [reply]
I am not sure but think it is the default download. I will try the Qt5 one.

**Reply by LeeBowditch** [reply]
UPDATE: So I have downloaded the QT5 version of Stellarium 1.1.1 and it works! It connected to the telescope right away with no issues at all. Thank you for all your help. Clear skies. Lee.

**Comment by jhenley57** [comment]
Mine is connected direct to rs-232 as com1, running Stelerium 1.1.1 with qt5, I have not tested to see if there is any action on the rs232 link, will test with breakout box soon , hopefully next week, but  meade autostar runs it and skytrack as well with no mods
________________________________

**Comment by faggio79** [comment]
Same problem here. I've a Skywatcher Synscan V5 hand controller, but the behavior is the same described for the Meade, the serial port is shown, but the control is always "stopped". Updated Stellarium to version 1.2.5 (QT6) but still not working. 

If i rollback to version 0.22 it works fine.

**Comment by jhenley57** [comment]
Where did you get 1.1.1 with qt5 version, I found 1.2 qt5 32 bit verson and 1.1.1 64 bit with qt6?

where can I get 64 bit 1.1.1 qt5 ?  can I delete qt6 files and put in qt5 files .dll etc?
________________________________

**Reply by gzotti** [reply]
> Where did you get 1.1.1 with qt5 version, I found 1.2 qt5 32 bit verson and 1.1.1 64 bit with qt6? where can I get 64 bit 1.1.1 qt5 ? 

stellarium.org, link "all releases". Meanwhile, try 1.2-qt5. 

> can I delete qt6 files and put in qt5 files .dll etc?

Sure way to failure.

**Reply by LeeBowditch** [reply]
Here is a direct link to 1.2 qt5 win64 version:
https://github.com/Stellarium/stellarium/releases/download/v1.2/stellarium-1.2-qt5-win64.exe

To uninstall there should be a uninstall app in the windows menu under Stellarium. If not you can go to the folder where it is installed and run unins000.exe

**Comment by jhenley57** [comment]
you are definately a prick!
________________________________

**Comment by jhenley57** [comment]
Thanks Lee!

This works very well, just installed it few 30 minutes ago and I really like the way it works.
Thanks so much for sending me the link. If I can ever help you with anything just let me
know.  Excellant fix, using windows 64 bit and Meade LX200 EMC

Jerry H.


________________________________
